kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -120,7 +120,9 @@ It is helpful to understand the following high-level flow before running the sam
    - If you models reference Kubernetes secrets, then include the following keys to the domain resource yaml file as needed:
    
      ```
-     configOverrideSecrets: [my-secret, my-other-secret]
+     configuration:
+       overrides:
+         secrets: [my-secret, my-other-secret]
      ```
 
    - In addition, specify the domain type in the domain resource yaml attribute. Valid values are `WLS`, `JRF`, and `RestrictedJRF`.
```

Comment:
I have a pending pull with a heavily updated README that conflicts.  See https://github.com/oracle/weblogic-kubernetes-operator/pull/1413/files